plugins/GSdx/GSSetting.cpp

```diff
@@ -57,9 +57,13 @@ const char* dialog_message(int ID, bool* updateText) {
 				"Works as a speedhack for: Steambot Chronicles.";
 		case IDC_SKIPDRAWHACK:
 		case IDC_SKIPDRAWHACKEDIT:
-			return "Skips drawing n surfaces completely. "
-				"Use it, for example, to try and get rid of bad post processing effects."
-				" Try values between 1 and 100.";
+		case IDC_SKIPDRAWOFFSET:
+		case IDC_SKIPDRAWOFFSETEDIT:
+			return "Completely skips drawing surfaces from the surface in the left box up to the surface specified in the box on the right.\n\n"
+				"Use it, for example, to try and get rid of bad post processing effects.\n"
+				"Step 1: Increase the value in the left box and keep the value in the right box set to the same value as the left box to find and remove a bad effect. Try values between 1 and 100.\n"
```

Comment:
The gui allows values up to 10000.

Reply:
I read it as a recommendation. Alternatively it could be removed.

BTW, you approved #2290 but the wording here hasn't changed. ;)

Reply:
>I read it as a recommendation. Alternatively it could be removed.

Ye I kinda guessed that, I say we remove it.

>BTW, you approved #2290 but the wording here hasn't changed. ;)

Thought there was no need for that since you're the one PRing it :)

Reply:
> Thought there was no need for that since you're the one PRing it :)

My point was that you've kinda approved something that you want changes to be made to, in which case it's ambiguous as to whether you've approved it or not.

Reply:
Ah, well in this case it's just a miniature nitpick but I'm fine with it being merged either way.